test/test_gesv.cc

```diff
@@ -91,6 +92,13 @@ void test_gesv_work(Params& params, bool run)
     params.gflops2();
     params.gflops2.name( "trs gflop/s" );
 
```

Comment:
Another issue: There's a conflict between the existing code using `time2` (above, for getrf and getrs) and the new timers using `time2` (below, for gesv). Ditto in `test_posv`. I can add a commit to handle this — harder to describe than just for me to do it.